api/test/unit/routes/routesHelper.test.js

```diff
@@ -61,7 +75,45 @@ describe('Policy Route helper ', function() {
 			});
 		});
 	});
-	
+
+	context('Create a policy and delete',function(){
+		beforeEach(function(done){
+			var mockRequest = {
+					body : fakePolicy,
+					params : { 'app_id' : '12348' }
+			};
+			var mockSchedulerResponse = {'statusCode':HttpStatus.OK };
+			routeHelper.createOrUpdatePolicy(mockRequest,mockSchedulerResponse,function(error,result){
+				done();
+			});
+		});
+		
+		it('should delete the policy with app_id 12348',function(done){
+			var mockRequest = {
+					params : { 'app_id' : '12348' }
+			};
+			var mockSchedulerResponse = {'statusCode':HttpStatus.OK };
+			routeHelper.deletePolicy(mockRequest,mockSchedulerResponse,function(error,result){
+				expect(error).to.be.null;
+				expect(result.statusCode).to.equal(200);
+				done();
+			});
+		});
+
+		it('should fail to delete the policy due to an internal server error',function(done){
```

Comment:
You didn't prove that it doesn't delete the policy, you just verify that you get a 500.